src/main/java/roomescape/application/dto/ReservationResponse.java

```diff
@@ -30,11 +33,17 @@ public LocalDate getDate() {
         return date;
     }
 
-    public LocalTime getTime() {
+    public TimeResponse getTime() {
         return time;
     }
 
     public static ReservationResponse from(final Reservation reservation) {
-        return new ReservationResponse(reservation.getId(), reservation.getName(), reservation.getDate(), reservation.getTime());
+        return new ReservationResponse(reservation.getId(), reservation.getName(), reservation.getDate(), TimeResponse.from(reservation.getTime()));
+    }
+
+    public static List<ReservationResponse> from(final List<Reservation> reservations) {
+        return reservations.stream()
+                .map(reservation -> new ReservationResponse(reservation.getId(), reservation.getName(), reservation.getDate(), TimeResponse.from(reservation.getTime())))
+                .collect(Collectors.toList());
```

Comment:
```suggestion
        return reservations.stream()
                .map(reservation -> new ReservationResponse(reservation.getId(), reservation.getName(), reservation.getDate(), TimeResponse.from(reservation.getTime())))
                .toList();
```

바로 toList()로 만들 수 있을 것 같습니다~